app/models/metrics/moves/counts_by_move_type.rb

```diff
@@ -2,31 +2,32 @@ module Metrics
   module Moves
     class CountsByMoveType
       include BaseMetric
+      include Moves
 
-      METRIC = {
-        label: 'Move counts by move type',
-        file: 'moves/counts_by_move_type',
-        interval: 5.minutes,
-        columns: {
-          name: 'move_type',
-          field: :itself,
-          values: Move.move_types.values,
-        },
-        rows: {
-          name: 'total',
-          field: :itself,
-          values: %w[total],
-        },
-      }.freeze
-
-      def initialize
-        setup_metric(METRIC)
+      def initialize(supplier: nil)
+        setup_metric(
+          supplier: supplier,
+          label: 'Move counts by move type',
+          file: 'counts_by_move_type',
+          interval: 5.minutes,
+          columns: {
+            name: 'move_type',
+            field: :itself,
+            values: Move.move_types.values << TOTAL,
+          },
+          rows: {
+            name: COUNT,
```

Comment:
talked in person but for posterity, in json files we are getting an extra column for counts example: 
```
"data": [
    {
      "count": "count",
      "court_appearance": 16151,
      "court_other": 1,
      "hospital": 4,
      "police_transfer": 3,
      "prison_recall": 1849,
      "prison_remand": 18,
      "prison_transfer": 9944,
      "video_remand": 2,
      "total": 27972
    }
  ]
```

Reply:
now fixed 👍